doc/onboarding.md

```diff
@@ -0,0 +1,197 @@
+## pre-setup
+
+Ensure everyone is added to https://github.com/orgs/nodejs/teams/collaborators
+
+
+## onboarding to nodejs
+
+### intros
+
+
+### **thank you** for doing this
+
+  * going to cover four things:
+    * local setup
+    * some project goals & values
+    * issues, labels, and reviewing code
+    * merging code
+
+
+### setup:
+
+  * notifications setup
+    * use https://github.com/notifications or set up email
+    * watching the main repo will flood your inbox, so be prepared
+
+
+  * git:
+    * make sure you have whitespace=fix: `git config --global --add core.whitespace fix`
+    * usually PR from your own github fork
+    * **See "Updating Node.js from Upstream"**
+    * make new branches for all commits you make!
+
+
+  * `#node-dev` on `chat.freenode.net` is the best place to be to interact with the TSC / CTC / other collaborators
+
+
+### a little deeper about the project
+
+  * collaborators are effectively part owners
```

Comment:
Removed in the second line. The first line without it isn't actually true per-se.